src/AggregateFunctions/Streaming/CountedValueMap.h

```diff
@@ -359,9 +315,21 @@ class CountedValueMap
         swap(rhs);
     }
 
+    int compare(const T & l, const T & r) const
+    {
+        const auto & key_comp = m.key_comp();
```

Comment:
This seems quite inefficient since we may need 2 compares to figure out greater, less or equal.